overview/demo/shell/default.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  pkgs,
+  apps ? null,
+}:
+
+let
+  inherit (lib)
+    getAttrFromPath
+    makeBinPath
+    splitString
+    isString
+    isList
+    ;
+
+  stringToPackage = str: getAttrFromPath (splitString "." str) pkgs;
+
+  getAppList =
```

Comment:
Especially this is unnecessary since we can just pass the packages directly from the module